### PR TITLE
Add `ConstantVariable` component

### PR DESCRIPTION
**Pull request by robinandeer**
## What

Implement new component to pass constant variables to a GraphQL mutation/query:

```xml
<GraphQLApi next="[[OfferSuccess]]">
  <Mutation>...</Mutation>
  <ConstantVariable key="input.payload[0].data.type"
                    value="SWEDISH_ACCIDENT"
                    as="string"
  >
  </ConstantVariable>
</GraphQLApi>
```

- Update tests to include new component.

## Why

We need this for new Quote Cart API. It also seems like something we want to have the option to do.

### CODE REVIEW

**Review by gustaveen — APPROVED** [review]
Cool! Just a question, what would `input.payload[0].data.type` resolve to in that example?

**Comment by robinandeer** [comment]
@gustaveen That would resolve to:

```json
{
  "input": {
    "payload": [{
      "data": {
        "type": "SWEDISH_APARTMENT"
      }
    }]
  }
}
```

It's using the same syntax as lodash set :)

File: src/Components/API/graphql.ts [hunk 1]
```diff
@@ -45,13 +46,18 @@ const isSingleVariable = (variable: Variable): variable is SingleVariable => {
...
-export const parseSingleVariable = (variable: SingleVariable, store: Store) => {
-  const value = store[variable.from]
-  if (value === undefined || value === null) {
-    return null
+const isConstantVariable = (
+  variable: Variable,
+): variable is ConstantVariable => {
+  if (variable.__typename === 'EmbarkAPIGraphQLConstantVariable') {
+    return true
   }
```
Comment (guilhermespopolin) [inline src/Components/API/graphql.ts:54]: [Just a suggestion]
```suggestion
const isConstantVariable = (
  variable: Variable,
): variable is ConstantVariable => variable.__typename === 'EmbarkAPIGraphQLConstantVariable'
```

File: src/Parsing/parseGraphQLApi.ts [hunk 2]
```diff
@@ -26,6 +30,13 @@ export interface MultiActionVariable {
   variables: Variable[]
 }
 
+export interface ConstantVariable {
+  __typename: string
```
Comment (guilhermespopolin) [inline src/Parsing/parseGraphQLApi.ts:34]: [Question] Shouldn't this be `__typename: ConstantVariable`? 🤔

**Review by guilhermespopolin — COMMENTED** [review]
Too late review. My bad

File: src/Parsing/parseGraphQLApi.ts (hunk 2 above)
Comment (robinandeer) [reply src/Parsing/parseGraphQLApi.ts:34]: @guilhermespopolin yeah I thought so at first but just followed the convention based on other types.

**Comment by hedvigoscar** [comment]
I guess I'm a bit late to the party, but couldn't you just inline the constant in the query body to achieve essentially the same result? 🤔

**Comment by robinandeer** [comment]
@hedvigoscar We could before but with quote cart API we build up more complex nested queries using #731 

At least I couldn't find another way around it.

**Pull request merged**